backend/services/twitter_client.py

```
@@ -52,33 +90,126 @@ def __init__(self):
+                            logger.info("Retrying with explicit None for OAuth 1.0a parameters")
+                            self.client = tweepy.Client(
+                                bearer_token=bearer_token,
+                                consumer_key=None,
+                                consumer_secret=None,
+                                access_token=None,
+                                access_token_secret=None,
+                                wait_on_rate_limit=True,
+                            )
+                            logger.info("Twitter client initialized with Bearer Token (OAuth 2.0)")
+                except Exception as init_error:
+                    logger.error(f"Failed to initialize tweepy.Client with bearer_token: {init_error}")
```

Ruff (0.14.5)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)